testsuite/regressiontests/issue-521.sli

```diff
@@ -0,0 +1,79 @@
+/*
+ *  issue-521.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+/* BeginDocumentation
+   Name: testsuite::issue-521 - *_hpc connections return local id instead of global id of target neuron
+
+   *_hpc connections should only return thread_local_target instead of target
+   to avoid confusion
+
+   FirstVersion: 04/2017
+   Author: Till Schumann
+ */
+
+(unittest) run
+/unittest using
+
+M_ERROR setverbosity
+
+is_threaded_with_openmp not { exit_test_gracefully } if
```

Comment:
Please replace this line with
```
skip_if_not_threaded
```
which has been added to master a few days ago. Travis currently fails without threading because we changed the behavior of `exit_test_gracefully` behind the scenes.